### PR TITLE
Bioperl update

### PR DESCRIPTION
**Pull request by pescobar**
updated bioperl-1.6.1 download urls

btw, BioPerl requires an urgent upgrade, this version was published in 2009

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar: this needs to be synced with develop

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ec52d9710d204d942c53 for a full test report.

File: easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.1-ictce-5.3.0-Perl-5.16.3.eb [hunk 1]
```diff
@@ -9,8 +9,8 @@ description = """Bioperl is the product of a community effort to produce Perl co
 
 toolchain = {'name': 'ictce', 'version': '5.3.0'}
 
-source_urls = ['http://www.cpan.org/modules/by-module/Bio/CJFIELDS/']
-sources = [SOURCE_TAR_GZ]
+source_urls = ['https://github.com/bioperl/bioperl-live/archive/']
+sources = ['bioperl-release-%s.tar.gz' % version.replace('.','-')]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.1-ictce-5.3.0-Perl-5.16.3.eb]: space after comma for a 3 above?

**Comment by fgeorgatos** [comment]
looks good

File: easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.923-ictce-5.5.0-Perl-5.18.2.eb [hunk 2]
```diff
@@ -9,8 +9,8 @@ description = """Bioperl is the product of a community effort to produce Perl co
 
 toolchain = {'name': 'ictce', 'version': '5.5.0'}
 
-source_urls = ['http://www.cpan.org/modules/by-module/Bio/CJFIELDS/']
-sources = [SOURCE_TAR_GZ]
+source_urls = ['https://github.com/bioperl/bioperl-live/archive/']
+sources = ['bioperl-release-%s.tar.gz' % version.replace('.', '-')]
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.923-ictce-5.5.0-Perl-5.18.2.eb]: hmm, this one isn't working (I added this)

Comment (boegel) [inline easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.923-ictce-5.5.0-Perl-5.18.2.eb]: should be `release-...`

Comment (boegel) [inline easybuild/easyconfigs/b/BioPerl/BioPerl-1.6.923-ictce-5.5.0-Perl-5.18.2.eb]: https://github.com/pescobar/easybuild-easyconfigs/pull/20

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/02ea97facaedc45f1be9 for a full test report.

**Comment by boegel** [comment]
Redownloaded verified as well. Going in, thanks @pescobar!

**Pull request merged**